test/e2e/config/docker.yaml

```diff
@@ -314,12 +314,12 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.29.2"
-  KUBERNETES_VERSION: "v1.30.0-rc.1"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.30.0"
```

Comment:
Oh this is the first time we're bumping KUBERNETES_VERSION_MANAGEMENT. I guess we missed it when we bumped to the RC?

We should probably do a follow-up to ensure this would also work (e.g. adding it to ci-e2e.sh). We can do that by just trying to use rc.1, but let's do this in a separate PR

Reply:
Testing here:

- https://github.com/kubernetes-sigs/cluster-api/pull/10459